macOS tests on x86 are failing

They are failing with the following traceback:

```
FAILED tests/isolation_provider/test_container.py::TestContainerTermination::test_linger_terminate - AssertionError: Expected 'kill' to not have been called. Called 1 times.
```

In previous episodes on this issue, we pushed a fix for this in https://github.com/freedomofpress/dangerzone/commit/04e286a762862ef4da197b672339329e4fb642dc, which was affecting `test_failed`, but this is a different issue.

I didn't manage to reproduce this error in our macOS intel machine.

--- a/tests/isolation_provider/base.py
+++ b/tests/isolation_provider/base.py
@@ -11,6 +11,7 @@ from dangerzone.isolation_provider import base
 # Timeout in seconds until the conversion sandbox starts.
 # Tests are really slow on macOS intel, hence the 5 minutes timeout.
 TIMEOUT_STARTUP = 60 * 5
+TIMEOUT_GRACE = 60
 
 
 @pytest.mark.skipif(
@@ -118,7 +119,7 @@ class IsolationProviderTermination:
         terminate_proc_spy = mocker.spy(provider, "terminate_doc_to_pixels_proc")
         popen_kill_spy = mocker.spy(subprocess.Popen, "kill")
 
-        with provider.doc_to_pixels_proc(doc) as proc:
+        with provider.doc_to_pixels_proc(doc, timeout_grace=TIMEOUT_GRACE) as proc:
             # We purposefully do nothing here, so that the process remains running.
             pass
 

